containers/package-builder/checkout-concent-version.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+DIRECTORY=$1
+
+# Check if a directory exists
+if [ -d "$DIRECTORY" ]; then
+    git fetch
+else
+    git clone /var/concent-data/concent/ "$DIRECTORY"
```

Comment:
Please make a shallow clone. There's no need to clone the whole repository if we only need the last revision. See the parameters that were used in the Ansible playbook.

Reply:
There is no parameter `--depth` in ansible playbook. This not work, because we clone repository locally.